ogr/ogrsf_frmts/adbc/ogradbcdataset.cpp

```diff
@@ -366,6 +436,127 @@ bool OGRADBCDataset::Open(const GDALOpenInfo *poOpenInfo)
         }
     }
 
+    const auto GetAsOpenOptionOrConfigOption = [poOpenInfo](const char *pszKey)
+    {
+        const char *pszVal =
+            CSLFetchNameValue(poOpenInfo->papszOpenOptions, pszKey);
+        if (pszVal)
+            return pszVal;
+        // CPLGetConfigOption("BIGQUERY_PROJECT_ID", nullptr);
```

Comment:
leftover?

Reply:
added "// Below comments are for scripts/collect_config_options.py" to explain what the purpose is